ax/service/ax_client.py

```diff
@@ -1197,9 +1198,10 @@ def get_model_predictions(
                 "argument to `get_model_predictions`."
```

Comment:
A few lines above, it says that `AxClient` only supports one-arm trials. Is that saying that `AxClient` doesn't support `BatchTrial`s? I didn't think that was the case. See e.g. here: https://github.com/facebook/Ax/blob/95881f4d0cdfc6dba525785d3e1ad7c3ecd8acd8/ax/service/ax_client.py#L1735

Reply:
I don't know about AxClient more generally but this seems to be a limitation of the `get_model_predictions` method. I'll clarify this in the docstring.